wlauto/resource_getters/standard.py

```diff
@@ -441,6 +441,12 @@ def resolve_resource(self, resource):
                             return asset
                         except ValueError as e:
                             self.logger.warning(e.message)
+        elif resource.name == 'executable':
+            device_abi = resource.owner.device.abi
```

Comment:
Should use `resource.platform` instead of the device's ABI. The owner may not have a device (and the resource requested may not be for the device).